### DISCUSSION TITLE
[BUG] How to upload test results to some server

### DISCUSSION THREAD

**Discussion by JaLe29** [ortoniKC/ortoni-report#38] [Q&A]
Is there any way how to upload test results to some my server?

I would like to create some test results UI to display results from hourly testing jobs.

**Comment by ortoniKC** [comment]
As of now it is standalone html file, 
You can upload to any server..

**Comment by JaLe29** [comment]
Yes, I can, but links to trace results don't work after uploading to the server, because the trace points to a playwright instance running locally. It might be nice to link to https://trace.playwright.dev/?trace=URL_TO_TRACE_FILE

**Comment by ortoniKC** [comment]
The trace file is required to view the trace, and it is stored in the ortoni-data folder, and trace viewer files are within the trace folder for local view. 
You can upload the entire ortoni-report folder to server, it should work fine. 
For ex, I have uploaded to ortoni.netlify.app

**Comment by JaLe29** [comment]
I would like to host my reports on following url:
https://xxxxxxxx/test-results/04-11-2024-09-37/index.html

But looks like that report can work only on base root domain like this: https://xxxxxxxx , links are broken.

**Comment by ortoniKC** [comment]
I hope you're using the latest version of 2.0.6

**Comment by JaLe29** [comment]
Of course. 

Look at my report:
https://evidia.fra1.cdn.digitaloceanspaces.com/test-results/04-11-2024-09-37/index.html

Try to click on View Trace button, there is a broken link... because I am using prefix `test-results/04-11-2024-09-37/` in my s3 storage

**Comment by ortoniKC** [comment]
Pls check with the default report and what the trace path looks like.

**Comment by ortoniKC** [comment]
It is fixed with the lastest version of V2.0.7, pls do check and let me know.
